src/api/data/helpers/helpers.go

```diff
@@ -106,3 +106,13 @@ func newestSemVer(v1 string, v2 string) (string, error) {
 	}
 	return v1, nil
 }
+
+// Int64Point converts an int64 to an *int64
+func Int64Point(n int64) *int64 {
```

Comment:
Super pedantic, and I don't really care, but `Ptr` is the usual abbreviation in Go for a pointer. Again, not at all a showstopper. Feel free to ignore.